README.md

```diff
@@ -38,9 +49,10 @@ Thus, training data is a list of size `N_years`, where each element is `ndarray`
 
 
 # Short Term Plan:
-0. Add `requirements.txt`. At the moment, dependencies are self-evident: just run `/notebooks/Demo.ipynb`;
-1. Review current data model (list of ndarrays);
-2. Set up data pipeline. E.g., select one scalar field for now;
-2. Evaluation metrics - WeatherBench;
-3. Implement a climatology model as benchmark;
-4. Uncertainty Quantification - DMD with bagging (Peter)
+[X]. Add `requirements.txt`. At the moment, dependencies are self-evident: just run `/notebooks/Demo.ipynb`;
+    - Now a pyproject.toml file
+[] Review current data model (list of ndarrays);
+[] Set up data pipeline. E.g., select one scalar field for now;
+[] Evaluation metrics - WeatherBench;
+[] Implement a climatology model as benchmark;
+[] Uncertainty Quantification - DMD with bagging (Peter)
```

Comment:
This list doesn't render properly, I think the list items should be like `- [ ] ...`